### ISSUE TITLE
WordPress Plugin

### ISSUE THREAD

**Issue by azrafe7** [pixedelic/Camera#26]
Hi,
let me say that you've managed to put up a great (and my favourite) js slideshow AND
&lt;!-- ~bad things here --&gt;
that I've got into troubles for trying to make it work with WordPress 3.5.1.
&lt;!-- ~end bad things here --&gt;

Don't know if you have scheduled a reissue of the plugin for the latest releases of WP (which I - and many others - would greatly appreciate)... in any case... surely I have NOT solved it fully (just kind of hacked it), but maybe this could help you.

Basically I'm just using the latest version of your js code (downloaded from your site) and have modified the camera_functions.php file to make it load only the needed files (skipping jQuery 1.7.1 and other unneeded scripts for the front-end).

Here's it, let me know if it's of use (https://dl.dropboxusercontent.com/u/32864004/camera-slideshow%20(hacked%20for%20wp%203.5.1).zip)

Cheers,
azrafe7
